plugins/Rockets/jsonPropertyMap.h

```diff
@@ -21,7 +21,20 @@
 #pragma once
```

Comment:
You should be the responsible author now for this file. Probably because it's not my code, but I don't understand much in here anymore.

Reply:
Sure, I think the responsible author is superfluous since you can see who has made exactly what changes using git blame.